resources/web/map/js/trajectory.js

```diff
@@ -81,35 +77,68 @@ function syncLayers() {
     }
 }
 
+const LNG_KM = 111.320, LAT_KM = 110.574;
+
+/**
+ * Helper method to create elliptical geojson data
+ * @param center {[lng: number, lat: number]}
+ * @param rX horizontal radius in kilometers of ellipse
+ * @param rY vertical radius in kilometers of ellipse
+ * @param points optional number of points to render ellipse, higher for smoothness
+ * @return {{geometry: {coordinates: [][], type: string}, type: string}}
+ */
+function createGeoJsonEllipse(center, rX, rY, points = 32) {
```

Comment:
the ellipse code will be useful if in future we want to plot the `MSG_POS_LLH_ACC:` or `MSG_POS_LLH_COV:` messages which allow break down of the horizontal accuracy into North/South and East/West components